PixArt/loader.py

```diff
@@ -76,6 +76,8 @@ def load_pixart(model_path, model_conf):
 		device=model_management.get_torch_device()
 	)
 
+	model_conf.unet_config['depth'] = sum(key.endswith('.scale_shift_table') for key in state_dict.keys())

```

Comment:
Overriding the blocks here is a bad idea, and using `.scale_shift_table` will always be off by one because the final layer also has a scale shift table entry.  I think just leave the loader as-is in this PR.

Reply:
The problem is that if you don't override it it messes up the generation for the larger models as it expects 28 layers not 42.

Reply:
Fixed using simple loader and an additional config in the standard loader.